README.md

```diff
@@ -57,14 +56,4 @@ deny_on_deprecation: true # note: this is set to true by default
 Kubernetes deprecation evolve over the time. As soon as new deprecations are
 added this policy will be updated.
 
-The versioning scheme of this policy follows this pattern:
-
-```
-<policy version>-k8sv<most recent version of kubernes known by the embedded deprecation rules>
-```
-
-For example, the release `0.1.0-k8sv1.26.0` of this policy knows about all the deprecation rules
-formulated up to Kubernetes release 1.26.0.
-
-The announcement of new deprecation rules for Kubernetes 1.27.0 would trigger
-then a release of this policy with the following version: `v0.1.1-k8sv1.27.0`.
+Currently, this policy is aware of deprecations introduced up to the Kubernetes **1.32.0** release.
```

Comment:
I would prefer it upwards in the readme, feels a bit hidden.

Reply:
done